src/main/scala/io/iohk/ethereum/db/storage/AppStateStorage.scala

```diff
@@ -27,6 +34,16 @@ class AppStateStorage(val dataSource: DataSource) extends TransactionalKeyValueS
   def getBestBlockNumber(): BigInt =
     getBigInt(Keys.BestBlockNumber)
 
+  def getBestBlockData(): BestBlockInfo =
+    BestBlockInfo( // FIXME default value for hash ?
+      get(Keys.BestBlockHash).map(v => ByteString(Hex.decode(v))).getOrElse(ByteString.empty),
+      getBigInt(Keys.BestBlockNumber)
+    )
+
+  def putBestBlockData(b: BestBlockInfo): DataSourceBatchUpdate =
```

Comment:
I wonder if the method would not have been better named: putBestBlock**Info** instead of **Data** to keep it coherent with the parameter type

Reply:
Completely agree. This is an oversight